packages/paste-website/src/pages/components/popover/index.mdx

```diff
@@ -298,6 +308,27 @@ To launch a Popover from a Badge component, use the PopoverBadgeButton component
   {badgeExample}
 </LivePreview>
 
+### PopoverFormPillButton
+
+This component should **only be used as part of the filter pattern**. It renders a FormPill and accepts all of its props, which are listed [on the Form Pill Group page](/components/form-pill-group#basic).
```

Comment:
suggestion: link to filter pattern

Reply:
This should be done as part of the filters patter PR. My reasoning for this is our links will fail (including tests) if we navigate to the new pattern and if we navigate to the existing pattern the popover is not currently used.

Reply:
makes sense! in the other docs page in this PR you added a (link coming soon!) so might want to either do that one here too or remove it completely in the other one, to stay consistent. make sure it's listed in the filter pattern ticket AC to update them!

Reply:
Great suggestion! Made that change